README.md

```diff
@@ -1,2 +1,4 @@
 # StickyNotes
 Application qui permet d'ajouter des post-its sur un chalkboard partagé.
+
+(pip install -r requirements.txt)
```

Comment:
N'hésitez pas à documenter l'installation du projet et comment on le lance avec un peu plus de détails.
[Certains](https://github.com/HE-Arc/ValEvasion#try-our-app-in-local) l'ont déjà fait. Je vous conseille de le faire également. En plus ça vous aide aussi si vous devez reconstruire votre projet après un nouveau `git clone` (en local comme en prod).

Reply:
Oui, on va le faire le plus vite possible. Merci